lib/Pool.js

```diff
@@ -14,6 +14,7 @@ const MAX_SIZE_DEFAULT = null;
 const SHRINK_DEFAULT = true;
 const CONNECTION_TIMEOUT_DEFAULT = 0;
 const LOGIN_TIMEOUT_DEFAULT = 0;
+const FETCH_ARRAY_DEFAULT = false; // RUC
```

Comment:
RUC?

Reply:
Initialism in my working copy. I thought I removed all of those but seem to have missed one.

Reply:
And corrected now.

Reply:
No problem. Figured it was something like that.